release_tester/tools/killall.py

```diff
@@ -14,17 +13,14 @@ def get_all_processes():
     for process in psutil.process_iter(['pid', 'name']):
         try:
             name = process.name()
-            if name.endswith('.exe'):
-                name = name[:-4]
-            if name == 'arangod':
+            if name.startswith('arangod'):
```

Comment:
switch with `arangodb` else this will match both.

Reply:
good catch! :+1: